source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+.. include:: ../references.txt
+
+.. iact_events:
+
+``EVENTS`` extension
+====================
+
+The ``EVENTS`` extension is a binary FITS table that contains an event list.
+Each row of the table provides information that characterises one event.
+The mandatory and optional columns of the table are listed below. In
+addition, a list of header keywords providing metadata is specified.
+Also here there are mandatory and optional keywords.
+The suggested extension name of the binary table is ``EVENTS``.
+
+
+Mandatory columns
+-----------------
+
+* ``EVENT_ID`` tform: ``1J``
+    * Event identification number at the DL3 level
+      (lower data levels could be different, see note below).
+* ``TIME`` tform: ``1D``, unit: s
+    * Time stamp of event in instrument specific MJD time reference. See the
+      header keywords ``MJDREFI`` and ``MJDREFF`` for the zero point of
+      the reference time.
+      See also the `OGIP event list`_ standard.
+* ``RA`` tform: ``1E``, unit: deg
+    * Reconstructed event Right Ascension (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
```

Comment:
I'm not happy with this DETX / DETY definition, I don't think anyone outside HESS knows what "nominal system" is, or that it's a good term to propagate. But this was just a side comment, I'm OK to put this in as-is and will (hopefully very soon) make a pull request for FOV coordinates where we can then discuss this.